inference-engine/src/vpu/myriad_plugin/myriad_config.cpp

```diff
@@ -95,6 +97,14 @@ IE_SUPPRESS_DEPRECATED_END
         { VPU_MYRIAD_CONFIG_VALUE(POWER_STAGE_NCES),   PowerConfig::STAGE_NCES },
     };
 
+    static const std::unordered_map<std::string, MovidiusDdrType> memoryTypes = {
+        { VPU_MYRIAD_CONFIG_VALUE(DDR_NOT_PROG), MovidiusDdrType::NOT_PROG },
```

Comment:
Could you please explain what is it `DDR_NOT_PROG` means?

Reply:
Something like _not set_, _auto_, _default_ 
`NOT_PROG ` I peeked it in the mdk) In this case, it means that the firmware define the type at runtime.

Reply:
In that case I suggest somethinkg like `AUTO`, `DEFAULT` or `RUNTIME` (probably it's better to align with other options if possible)